test/integration_test/extender_test.go

```diff
@@ -146,3 +151,65 @@ func driverNodeErrorTest(t *testing.T) {
 
 	destroyAndWait(t, ctxs)
 }
+
+func pvcOwnershipTest(t *testing.T) {
+	ctxs, err := schedulerDriver.Schedule(generateInstanceID(t, "ownershiptest"),
+		scheduler.ScheduleOptions{AppKeys: []string{"mysql-repl-1"}})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout, defaultWaitInterval)
+	require.NoError(t, err, "Error waiting for pod to get to running state")
+
+	scheduledNodes, err := schedulerDriver.GetNodesForApp(ctxs[0])
+	require.NoError(t, err, "Error getting node for app")
+	require.Equal(t, 1, len(scheduledNodes), "App should be scheduled on one node")
+
+	volumeNames := getVolumeNames(t, ctxs[0])
+	require.Equal(t, 1, len(volumeNames), "Should have only one volume")
+
+	verifyScheduledNode(t, scheduledNodes[0], volumeNames)
+
+	for _, spec := range ctxs[0].App.SpecList {
+		if obj, ok := spec.(*storage_api.StorageClass); ok {
+			if err := k8s.Instance().DeleteStorageClass(obj.Name); err != nil {
+				require.NoError(t, err, "Error deleting storage class for mysql.")
+			}
+		}
+		if obj, ok := spec.(*v1.PersistentVolumeClaim); ok {
+			updatePVC, _ := k8s.Instance().GetPersistentVolumeClaim(obj.Name, obj.Namespace)
+			if _, hasKey := updatePVC.Annotations["storage-provisioner"]; hasKey {
```

Comment:
The annotation is `volume.beta.kubernetes.io/storage-provisioner`. Define a const for that and use that here

Reply:
The annotation in the const is incorrect